### ISSUE TITLE
Creating A New Starter Should Include Acidification

### ISSUE THREAD

**Issue by tbpassin** [hendricius/the-sourdough-framework#381]
We've known from the mid-2000s that acidifying the starter-to-be at the start makes the whole process much faster and less prone to trouble.  See Debra Winks on TFL: https://www.thefreshloaf.com/node/10856/pineapple-juice-solution-part-1

Also see my recent post on TFL, which reports on my creating a new starter that was ready to go in two days:

[New Starter In Two Days - Wow!](https://www.thefreshloaf.com/node/74455/new-starter-two-days-wow)

**Comment by hendricius** [comment]
@tbpassin nice article there. Do you think your starter was ready so fast because you used the pickle brine? You effectively used the spores from your pickles to create a new starter 😎.

**Comment by tbpassin** [comment]
Yes I certainly do.  And no. I'm sure it wasn't spores from the pickle brine, it was the acidity.  With a pH that low, yeast growth in the pickles was very suppressed. If you go back to that thread, you will see a follow-up comment by Debra Winks.  She has gotten the same 2-day creation of a new starter using a bit of citric acid to get a similarly low initial pH.

When the low pH liquid is mixed with flour, apparently the dormant yeast spores can wake up and become active.  With the acidic liquid, you bypass those early days that plague many efforts, and avoid the mixture getting stuck growing unwanted or disease organisms.

**Comment by hendricius** [comment]
Very interesting experiment. I personally had a starter ready after 3 days, without changing acidity. That could also come from the high contamination with starter microbes in my kitchen. It would be nice to do a bit more experimentation here. Any suggestions on how to incorporate this into the book?

**Comment by tbpassin** [comment]
Maybe you haven't seen all the threads on The Fresh Loaf begging for help with new starters.  They didn't get a new one going in three days, even using whole grains. The only one I got going that fast, before the pickle brine, was using beaten rice flakes.  And like you I suspect that the high level of yeast in my working area may have helped it along.

I will think about possible changes to that section.

**Comment by DocDough** [comment]
It is perhaps worthwhile going back to Debra Wink's early posts on TFL about the pineapple juice solution which showed the benefit of initial acidification in the starter-to-be by adding pineapple juice.

It turns out that you can actually initiate a starter using commercial yeast (to my great surprise when I tried it in an attemp to demonstrate that it can't be done).  It appears that the pH suppression that results from the carbonic acid created by the CO2 produced by the commercial yeast is enough to suppress some of the important spoilage bacteria. And the commercial yeast is in turn suppressed by the lactic acid that is eventually produced by the sourdough lactobacteria.

**Comment by pcn** [comment]
>  That could also come from the high contamination with starter microbes in my kitchen.

That could be it. I've been a pretty consistent sourdough baker for the past 20 years, and the most difficult starter I've ever tried to get going was in my most recent apartment which was completely refurbished. It really did need pineapple juice and whole wheat flour to get a spark of life. Regular commercial unbleached white flour just seemed to not go anywhere. 

In contrast, when I lived in brazil where my starter got going from flour from plastic bags that are very slightly perforated and so are in some sense constantly in contact with a more humid environment, and always just slightly more contaminated than the typical supermarket shelf in the US. This was the fastest starter I ever had.

So I'm chiming in because your advice of "just leave it out for a few days" will work a lot of the time, but a lot of the time it won't too. I'd encourage you to add some material about this.
